packages/x-charts/src/LineChart/LineHighlightElement.tsx

```diff
@@ -43,11 +43,11 @@ const HighlightElement = styled('circle', {
   name: 'MuiHighlightElement',
   slot: 'Root',
   overridesResolver: (_, styles) => styles.root,
-})<{ ownerState: LineHighlightElementOwnerState }>(({ ownerState }) => ({
-  transform: `translate(${ownerState.x}px, ${ownerState.y}px)`,
-  transformOrigin: `${ownerState.x}px ${ownerState.y}px`,
-  fill: ownerState.color,
-}));
+})<{ ownerState: LineHighlightElementOwnerState }>({
+  transform: `translate(var(--x)px, var(--y)px)`,
+  transformOrigin: `var(--x)px var(--y)px`,
```

Comment:
```suggestion
  transform: `translate(var(--x), var(--y))`,
  transformOrigin: `var(--x) var(--y)`,
```

concatenate won't work in CSS, please move to inline style.